packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -212,9 +212,14 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
           clientType: options?.clientType || defaultClientType,
           clientVersion: providedClientVersion || defaultSDKClientVersion,
           ...versionMetadata,
-          localNetworkPrefix:
+          publicNetworkPrefix:
             // @ts-ignore
             anonymizeIPAddress(this.webex.meetings.geoHintInfo?.clientAddress) || undefined,
+          localNetworkPrefix:
+            anonymizeIPAddress(
+              // @ts-ignore
+              this.webex.meetings.getActiveWebrtcMeeting()?.getDiagnosticsLocalIp()
```

Comment:
don't use getActiveWebrtcMeeting() - this is only meant for debugging. You already have a meetingId as the argument of this function here, so

Reply:
could we potentially extract this as a separate func?

Reply:
Im not sure I see the value here? it's only called once and inlined here same as publicNetworkPrefix

Reply:
@marcin-bazyl : done